internal/descriptor/registry.go

```diff
@@ -87,6 +87,9 @@ type Registry struct {
 	// has no HttpRule annotation.
 	warnOnUnboundMethods bool
 
+	// optionalNullable specifies whether Proto3 Optional fields should be marked as x-nullable.
+	optionalNullable bool
```

Comment:
Can we name this field and related helpers `proto3OptionalNullable` to make it more explicit?

Reply:
Yep, done. I was debating this myself.

Reply:
Yep, done. I was debating this myself.